bindings/python/templates/model.mustache

```diff
@@ -0,0 +1,237 @@
+# coding: utf-8
```

Comment:
Original file can be found here: https://github.com/swagger-api/swagger-codegen/blob/v2.4.26/modules/swagger-codegen/src/main/resources/python/model.mustache

Changes consist of removing all `noqa : E501` from model property getters and setters